ui-v2/tests/helpers/type-to-url.js

```diff
@@ -2,22 +2,30 @@ export default function(type) {
   let url = null;
   switch (type) {
     case 'dc':
-      url = '/v1/catalog/datacenters';
+      url = ['/v1/catalog/datacenters'];
       break;
     case 'service':
-      url = '/v1/internal/ui/services';
+      url = ['/v1/internal/ui/services', '/v1/health/service/'];
       break;
     case 'node':
-      url = '/v1/internal/ui/nodes';
-      // url = '/v1/health/service/_';
+      url = ['/v1/internal/ui/nodes'];
       break;
     case 'kv':
       url = '/v1/kv/';
       break;
     case 'acl':
-      url = '/v1/acl/list';
-      // url = '/v1/acl/info/_';
+      url = ['/v1/acl/list'];
       break;
   }
-  return url;
+  return function(actual) {
+    if (url === null) {
+      return false;
+    }
+    if (typeof url === 'string') {
+      return url === actual;
+    }
```

Comment:
If you are still supporting url as a string, why make everything an array in the switch case?

Reply:
Hmm, yeah, `kv` is still a string, but even then, why did I leave it as a string and not change it to an array, will have another look at that. This area prime refactor area in a week or so, I've already done a feature bump to the `api-double` that it depends on - and this file and the `set-cookies` one have always been first in the queue to try to get rid of/improve, will have a quick look now though.